dynamic_env_validator.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+import yaml
+from pydantic import EmailStr, Field, create_model
+from pydantic_settings import BaseSettings
+
+
+class FileStr(str):
+    @classmethod
+    def __get_validators__(cls):
+        yield cls.validate
+
+    @classmethod
+    def validate(cls, value, field):
+        if not os.path.exists(value):
```

Comment:
exists is true for both directory and file...

Reply:
Done